src/coral_models/utils.py

```diff
@@ -0,0 +1,30 @@
+"""General utility functions."""
+
+import warnings
+
+import datasets.utils.logging as ds_logging
+import transformers.utils.logging as hf_logging
+from datasets.utils import disable_progress_bar
+
+
+def block_terminal_output() -> None:
+    """Blocks undesired terminal output."""
+
+    # Ignore miscellaneous warnings
+    warnings.filterwarnings("ignore", category=UserWarning)
+
+    # Disable `datasets` logging
+    ds_logging.set_verbosity_error()
+
+    # Disable the tokeniser progress bars
+    disable_progress_bar()
+
+
+class ignore_transformers_output:
```

Comment:
Why do we use snake_case instead of PascalCase for class name?

Reply:
From my comment from below:

> It seems as though context managers specifically are usually written in snake case, regardless of their implementation being function- or class based. Examples include warnings.catch_warnings and open (when opening files). Some examples can be found in [PEP 343](https://peps.python.org/pep-0343/#examples).